src/libraries/System.Text.Json/src/System/Text/Json/Serialization/WriteStackFrame.cs

```diff
@@ -20,8 +20,8 @@ internal struct WriteStackFrame
         /// The original JsonPropertyInfo that is not changed. It contains all properties.
         /// </summary>
         /// <remarks>
-        /// For objects, it is either the policy property for the class or the current property.
-        /// For collections, it is either the policy property for the class or the policy property for the current element.
+        /// For objects, it is either the "type property" for the class or the current property.
+        /// For collections, it is either the "type property" for the class or the "type property" for the current element.
         /// </remarks>
         public JsonPropertyInfo? DeclaredJsonPropertyInfo;
```

Comment:
Since this is a "special" `JsonPropertyInfo`, maybe it should be renamed to reflect that as well.